stream-chat-android-client/src/debug/java/io/getstream/chat/android/client/di/NetworkFlipper.kt

```diff
@@ -0,0 +1,8 @@
+package io.getstream.chat.android.client.di
+
+import com.facebook.flipper.plugins.network.FlipperOkhttpInterceptor
+import com.facebook.flipper.plugins.network.NetworkFlipperPlugin
+
+public val networkFlipper: NetworkFlipperPlugin = NetworkFlipperPlugin()
+
+public fun flipperInterceptor(): FlipperOkhttpInterceptor = FlipperOkhttpInterceptor(networkFlipper)
```

Comment:
Any reason for having this in a separate file?

Reply:
`networkFlipper` is being used both in ChatModule and `ExtraDependenciesImpl`. It would be weird to put everything inside ChatModule